docs/docs/action-server.mdx

```diff
@@ -6,7 +6,4 @@ description: Check out the API docs for open source chatbot framework Rasa's act
 hide_table_of_contents: true
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import Redoc from '@site/src/components/redoc';
-
-<Redoc specUrl={useBaseUrl("/spec/action-server.yml")} />
+See API link in header.
```

Comment:
is this intended? nothing in header

Reply:
yes the header contains a link to the API page, this is where redoc will live, so it can occupy the full page width

Reply:
My intention is to flag these files for @erohmensing i.e. that they need to be updated or removed

Reply:
but in the meantime, it means that the deployed doc won't have the API spec available. What about simply adding a comment in the header for Ella? Something like `FIXME: @erohmensing to handle this change` or something like that?

Reply:
OK reverted this too. Will ping ella when the new solution is up